Interview/Extension/String+Localizable.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  String+Localizable.swift
+//  Interview
+//
+//  Created by Jadië Oliveira on 14/09/23.
+//
+
+import Foundation
+
+extension String {
+    init(localizedKey:String) {
+        let initText = NSLocalizedString(localizedKey, tableName: "Localizable", bundle: .main, value: "", comment: "")
```

Comment:
Dá pra usar a versão menor do NSLocalizedString, já que o arquivo de string está com o nome default reconhecido pelo Xcode.
```suggestion
        let initText = NSLocalizedString(localizedKey, comment: "")
```